core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -295,6 +295,8 @@
   TSERV_READ_AHEAD_MAXCONCURRENT("tserver.readahead.concurrent.max", "16", PropertyType.COUNT,
       "The maximum number of concurrent read ahead that will execute. This effectively"
           + " limits the number of long running scans that can run concurrently per tserver."),
+  TSERV_READ_AHEAD_PREFIX("tserver.readahead.concurrent.queue.", null, PropertyType.PREFIX,
```

Comment:
Would be nice to use a more generic prefix.   I was thinking the only thing that could be configured from the scanner is which set of predefined resources to use.  A scanner could have a method like `useResources(String name)`. If some configs like the following were set, then `scanner.useResources("default")` or `scanner.useResources("slowlane")` could be called.

```
tserver.resources.scan.default.numThreads=32
tserver.resources.scan.default.threadPrio=10
tserver.resources.scan.default.queueType=....
#This is the max amount of data all threads in the group can read per sec
tserver.resources.scan.default.rateLimit=1024MB/S
tserver.resources.scan.slowlane.numThreads=4
tserver.resources.scan.slowlane.threadPrio=20
tserver.resources.scan.slowlane.queueType=....
tserver.resources.scan.slowlane.rateLimit=100MB/S
```

Also, we could define a new per table setting that defines the default resources scans against that table should use.

```
config -t myTable -s table.resources.scan.use=slowlane
```

This would make a all scans against myTable use resources from slowlane by default.